packages/@lightningjs/ui-components/src/components/InlineContent/InlineContent.js

```diff
@@ -360,6 +359,28 @@ export default class InlineContent extends Base {
         text: text.text || text
       }
     };
+    if (textOverrideStyles?.textDecoration === 'line-through') {
+      const textWidth = measureTextWidth({
+        ...this.style.textStyle,
+        ...textOverrideStyles,
+        text: text.text || text
+      });
+      const strikethroughLine = {
+        rect: true,
+        w: textWidth - 2,
+        color: textOverrideStyles?.textColor || 0xffffffff,
+        h: textComponent.h * 0.08,
```

Comment:
1. the width and the height are both being calculated with some hardcoded values, are we cool with that?
2. should the fallback color be derived from a theme value?

Reply:
1. @shanteaustin Do we need the 2 being subtracted? I think the height is a good callout, we can pull the 0.08 out into a style.
2. I don't think it hurts to point to a token by default. At least the strikethrough would be in line with the user's theme.

Reply:
1. @ImCoolNowRight The line seemed at bit long to me but after a couple tweaks it's not so bad so I will remove that. When you say pull the 0.08 into a style do you mean adding it to the theme object in InlineContent.styles? 
2. Can you give me an example of what you're looking for so I can make sure I know for sure.

Reply:
1. We have a `.styles.js` file for each component that has the styles used by it. We're suggesting adding a new style property to it, something like `strikethroughRatio`, that would be set to 0.08 by default, and then we reference that style instead of the hard coded number. That way, users can pass something else in if they want to.
2. I think textNeutral works!